packages/client/src/store/slices/forumSlice.ts

```diff
@@ -0,0 +1,72 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { TTopic } from '@/types/forumDataTypes';
+// import forumApi from '@/api/ForumApi';
+
+type TForum = {
+    forumTopics: TTopic[] | null;
+    topicComments: [] | null;
+};
+
+export type TState = {
```

Comment:
Мб стоит лучше наименовать 
```suggestion
export type ForumState = {
```

Reply:
done